demo/sticky.html

```diff
@@ -131,6 +149,32 @@ <h1>${firstLetter}</h1>
     }
     document.querySelector('#swapButton')
         .addEventListener('click', swapScrollers);
+
+    let perfDemoModeState = false;
+
+    /**
+     * Swaps between performance demo mode and not. In this mode we
+     * should the FPS and we do an animated resize of the contacts list.
+     */
+    function perfDemoMode() {
+      let fpsDisplayStyleDisplay;
+      let scrollContainerClass;
+      if (perfDemoModeState === false) {
```

Comment:
Do `if (perfDemoModeState) { ... } else { ... }`

Reply:
yeah, no idea what I was doing there. Anyway the code is a lot simpler now.